### PR TITLE
feat: add touch icons

### PR DESCRIPTION
**Pull request by anglepoised**
Uses a combination of [Evil Martians: How to Favicon in 2021](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs) article and a selection of assets prepared by @isrob last year.

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -6,8 +6,16 @@ module.exports = function (eleventyConfig) {
...
   });
 
+  // Copy assets
   eleventyConfig.addPassthroughCopy("./src/assets/img");
 
+  // Copy favicons and touch icons
+  eleventyConfig.addPassthroughCopy({
+    "./src/manifest.webmanifest": "manifest.webmanifest",
+    "./src/favicon.ico": "favicon.ico",
+    "./src/icon.svg": "icon.svg",
```
Comment (monospaced) [inline .eleventy.js:16]: Not seeing `.png` here, and also looks like the pngs wouldn't be captured by the `assets/img` path?

Comment (monospaced) [inline .eleventy.js:16]: Not seeing `.png` here, and also looks like the pngs wouldn't be captured by the `assets/img` path?

Comment (anglepoised) [reply .eleventy.js]: I went lazy and changed to using [Passthrough by File Extension](https://www.11ty.dev/docs/copy/#passthrough-by-file-extension) approach (besides the stuff from `node_modules` which needs a target mapped).

Comment (anglepoised) [reply .eleventy.js]: `.png` were actually copying before because they were listed in that `templateFormats` block but... not intuitive at all 😬

**Review by monospaced — APPROVED** [review]
boom!

**Pull request merged**